### PR TITLE
Feat/remove skillshome from flow

### PR DESCRIPTION
**Pull request by ErikKimsey**
* Created data model based on changes star asked for last week.
* Refactored a bit according to said changes (e.g., removing Skills Home)
* Adjusted some styles to accomodate the changes to Chains home, that star wants.

### CODE REVIEW

File: components/Chain/ScorecardListItem.tsx [hunk 1]
```diff
@@ -6,15 +6,15 @@ import { MasteryIcons } from "../../styles/MasteryIcons";
 import { AntDesign } from "@expo/vector-icons";
 
 type Props = {
-	id: string;
-	title: string;
-	score: number;
+	attemps: {}[];
```
Comment (danfunk) [inline components/Chain/ScorecardListItem.tsx:9]: attemps?  Is that a spelling mistake?

Comment (danfunk) [reply components/Chain/ScorecardListItem.tsx:9]: Looks like a bizarre data structure {}[]

**Review by danfunk — CHANGES_REQUESTED** [review]
Just a couple of questions there at the top.

File: components/Chain/ScorecardListItem.tsx (hunk 1 above)
Comment (ErikKimsey) [reply components/Chain/ScorecardListItem.tsx:9]: @danfunk Yes, misspelling.

It's Typescript's indication of an array of objects.

**Pull request merged**